test/browser/features/resource-load-spans.feature

```diff
@@ -0,0 +1,30 @@
+Feature: Resource Load Spans
+
+  Scenario: Resource load spans are automatically instrumented
+    Given I navigate to the test URL "/resource-load-spans"
+    And I wait to receive a sampling request
+
+    When I click the element "end-span"
+    And I wait to receive 1 trace
+
+    # Custom span (parent)
+    Then the trace payload field "resourceSpans.0.scopeSpans.0.spans.2.name" equals "[Custom]/resource-load-spans"
+    And the trace payload field "resourceSpans.0.scopeSpans.0.spans.2.spanId" is stored as the value "parent_span_id"
+
+    # App bundle
```

Comment:
depending on browser versions, yes! we should get some.

Reply:
considering these spans are already covered in this feature and will be incredibly difficult to test in the page load span feature, I think this might be a bit redundant.